install_config/registry/accessing_registry.adoc

```diff
@@ -272,3 +272,95 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
+
+[[access-reading-and-writing-image-signatures]]
+=== Reading and Writing Image Signatures
+The {product-title} integrated Docker Registry provides an "extensions" endpoint that
+allows you to write and read image signatures. The image signatures are stored in
+{product-title} API, but this endpoint allows you to read and write them without
+having direct access to the {product-title} API endpoint.
+
+[NOTE]
+====
+This endpoint is experimental and not supported by the upstream Docker Registry project.
+See https://docs.docker.com/registry/spec/api/[upstream API documentation] for general
+information about the docker registry API.
+====
+
+==== Writing Image Signatures
+In order to add a new signature to the Image, users can use the HTTP `PUT` method together
+with the JSON payload sent the extensions endpoint:
+
+====
+----
+PUT /extensions/v2/{namespace}/{name}/signatures/sha256:{digest}
+----
+====
+
+====
+----
+$ curl -X PUT --data @signature.json http://<user>:<token>@<registry-endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
+----
+====
+
+The JSON payload with the signature content should have the following structure:
+
+====
+----
+{
+  "version": 2,
+  "type":    "atomic",
+  "name":    "sha256:4028782c08eae4a8c9a28bf661c0a8d1c2fc8e19dbaae2b018b21011197e1484@cddeb7006d914716e2728000746a0b23",
+  "content": "<base64 encoded signature>",
+}
+----
+====
+
+The `name` field contains the name of the image signature, which has to be unique and in
+the following format: `sha256:<digest>@<name>`. The `name` has to be 32 character long.
```

Comment:
Explain that the first part is image name and the second is signature name. And that the first part must match the `<digest>` parameter of the endpoint.